onnx/version_converter/convert.h

```diff
@@ -396,6 +396,7 @@ class DefaultVersionConverter : public BaseVersionConverter {
     registerAdapter(std::make_unique<CompatibleAdapter>("Pow", OpSetID(11), OpSetID(12)));
     registerAdapter(std::make_unique<CompatibleAdapter>("ReduceMax", OpSetID(11), OpSetID(12)));
     registerAdapter(std::make_unique<CompatibleAdapter>("ReduceMin", OpSetID(11), OpSetID(12)));
+    registerAdapter(std::make_unique<CompatibleAdapter>("SequenceInsert", OpSetID(11), OpSetID(12)));
```

Comment:
I see only opset 11 version of [SequenceInsert](https://onnx.ai/onnx/operators/onnx__SequenceInsert.html#) ... so, why do we need an adapter?

Reply:
Same for below